lib/ansible/vars/manager.py

```diff
@@ -518,7 +518,7 @@ def _get_delegated_vars(self, play, task, existing_variables):
             items = [None]
 
         delegated_host_vars = dict()
-        item_var = getattr(task.loop_control, 'loop_var', 'item')
+        item_var = getattr(task.loop_control, 'loop_var', 'item') or 'item'
```

Comment:
you should not need `or item` as the getattr already defaults to 'item'

Reply:
It doesn't default to `'item'` if `loop_var` is `None` though, hence `or 'item'`, no?

Reply:
Wouldn't it make more sense to do?:
```
-    _loop_var = FieldAttribute(isa='str')
+    _loop_var = FieldAttribute(isa='str', default='item')
```

Reply:
Oh, it would, didn't think of that. Will fix the PR, thanks!